src/ast/groovy/org/grails/plugin/cache/compiler/CacheableTransformation.groovy

```diff
@@ -53,6 +53,13 @@ class CacheableTransformation extends AbstractCacheTransformation {
     @Override
     protected Expression buildDelegatingMethodCall(SourceUnit sourceUnit, AnnotationNode annotationNode, ClassNode classNode, MethodNode methodNode, MethodCallExpression originalMethodCallExpr, BlockStatement newMethodBody) {
 
+        boolean isControllerClass = classNode.name.endsWith('Controller')
+        boolean isServiceClass = classNode.name.endsWith('Service')
+
+        if (isControllerClass && !isServiceClass) {
```

Comment:
I am confused about the `if (isControllerClass && !isServiceClass)` clause.  I think that says if the class name ends in 'Controller' and doesn't end in 'Service', then return the original method.  What is the point of checking if the class name ends in 'Service' if the condition only applies if the class name ends in 'Controller'?